src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,93 @@
+import { Outlet } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PersonInfo } from '../PersonInfo/PersonInfo';
+
+export const PeoplePage = () => {
+  const [peopele, setPeople] = useState<Person[] | null>([]);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  useEffect(() => {
+    setLoading(true);
```

Comment:
why do we need to set this isLoading === true for componentDidMount, you can set true as initial value to the state